src/applications/check-in/components/AppointmentDisplay/AppointmentMessageVaos.jsx

```diff
@@ -84,7 +84,7 @@ const AppointmentMessageVaos = props => {
         if (Number.isNaN(appointmentDateTime.getTime())) {
```

Comment:
I struggle with this kind of refactor work in a ticket like this. I didn't write any of this code in the scope of task of this PR, so I hesitate to refactor it here since it seems out of scope. At the same time I am touching and reusing the existing code and choosing to do other refactors here and there. For this one, I think it is best left to tech debt since this logic is also happening in other place as well.

Reply:
Would like to hear @acrollet's thoughts here too

Reply:
Good idea lets bring this up in our app review meeting